pose_detector/DetectedObject[] detected_objects